addons/sourcemod/scripting/vsh/bosses/boss_seeldier.sp

```diff
@@ -91,6 +72,10 @@ methodmap CSeeldier < SaxtonHaleBase
 		for (int i = 0; i < iLength; i++)
 		{
 			int iClient = aValidMinions.Get(i);
+			
+			SaxtonHaleBase minion = SaxtonHaleBase(iClient);
```

Comment:
Could've just moved `boss` variable up here without needing to call `SaxtonHaleBase` on same client twice